lib/kafo/kafo_configure.rb

```diff
@@ -143,7 +143,7 @@ def initialize(*args)
       # so we limit parsing only to app config options (because of --help and later defined params)
       parse clamp_app_arguments
       parse_app_arguments # set values from ARGS to config.app
-      Logging.setup(verbose: config.app[:verbose])
+      Logging.setup(verbose: config.app[:verbose]) unless ARGV.include?('--help') || ARGV.include?('--full-help')
```

Comment:
```suggestion
      Logging.setup(verbose: config.app[:verbose]) unless ARGV.any? { |option| ['--help', '--full-help'].include? option }
```

I think we may want to have some Hooks suppress this output as well. This is logically equivalent, but easier to expand on.